examples/all-clusters-app/cc13x2x7_26x2x7/main/include/CHIPProjectConfig.h

```diff
@@ -117,4 +117,8 @@
  */
 #define CHIP_IM_MAX_NUM_WRITE_HANDLER 1
 
```

Comment:
Add Blocking comment until https://github.com/CHIP-Specifications/chip-test-plans/pull/2503 is merged